third_party/move/move-prover/src/cli.rs

```diff
@@ -67,7 +69,8 @@ pub struct Options {
     pub skip_attribute_checks: bool,
     /// Whether to use compiler v2 to compile Move code
     pub compiler_v2: bool,
-
+    /// The language version to use
+    pub language_version: Option<LanguageVersion>,
```

Comment:
nit: standardize with the rest of the code:
`maybe_language_version` or `optionjal_language_version`?

Also the comment suggest that the language version to use will be provided, what if its None?

Reply:
Actually all over the code (including Aptos CLI) we typically do not use `optional_...`. Changing this now would be inconsistent with 1/2 dozen other places which already existed before this PR. I personally think its OK to not encode this in the variable name.